src/sasl/cyrsasl_external_verification.erl

```diff
@@ -0,0 +1,47 @@
+%%%=============================================================================
+%%% @copyright (C) 1999-2019, Erlang Solutions Ltd
+%%% @author Denys Gonchar <denys.gonchar@erlang-solutions.com>
+%%% @doc cyrsasl external verification backend
+%%%
+%%% this module is added for demo & testing purposes only.
```

Comment:
It would be nice to have a short description how the verification implemented in this module is different from the standard modes built into `cyrsasl_external`.

Reply:
Something like:
```erlang
%%% This is experimental prototype module for demo & testing purposes only.
%%% It allows to use external verification service.
```
?

Reply:
No, more like how it verifies the credentials exactly.

Reply:
done